src/app/[locale]/dashboard/_components/app-sidebar.tsx

```
@@ -0,0 +1,200 @@
+"use client";
+import {
+  CalendarHeart,
+  ClipboardList,
+  EllipsisVertical,
+  Flower,
+  LayoutDashboard,
+  LogOut,
+  Package,
+  User,
+} from "lucide-react";
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarFooter,
+  SidebarGroup,
+  SidebarGroupContent,
+  SidebarHeader,
+  SidebarMenu,
+  SidebarMenuButton,
+  SidebarMenuItem,
+} from "@/components/ui/sidebar";
+import Image from "next/image";
+import logo from "../../../../../public/assets/Images/logo 1.png";
+
+import { Link, usePathname } from "@/i18n/navigation";
+import { signOut, useSession } from "next-auth/react";
+import { Button } from "@/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { useTranslations } from "next-intl";
+
+export function AppSidebar() {
+  // session
+  const { data: session } = useSession();
+  // Translations
+  const t = useTranslations();
+
+  // Variables
+  const firstName = session?.user.firstName ?? "Mohamed";
```

There’s no need to set a default username here.
If there’s no session or user data, the user shouldn’t be able to access the dashboard in the first place, so rendering a default name adds no value and could be misleading.